packages/contracts-bedrock/src/Safe/LivenessGuard.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Safe } from "safe-contracts/Safe.sol";
+import { BaseGuard, GuardManager } from "safe-contracts/base/GuardManager.sol";
+import { ModuleManager } from "safe-contracts/base/ModuleManager.sol";
+import { SignatureDecoder } from "safe-contracts/common/SignatureDecoder.sol";
+import { Enum } from "safe-contracts/common/Enum.sol";
+
```

Comment:
Should we prefer custom errors over require on new contracts?

Reply:
Hmmm, good question.

I used them in DelayedVetoable, but not in the `SuperchainConfig`, because I was aiming to maintain consistency with the `SystemConfig` and `ProtocolVersions`. 

The style guide is actually fairly opinionated, that we should use strings.

https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/STYLE_GUIDE.md#L28-L34

Reply:
Not really a concern as there are no errors in this contract.